### ISSUE TITLE
Elden ring still not working:( - gtx 960m

### ISSUE THREAD

**Issue by lowkeyrenato** [tuffee88/d3d12ProxyEdrDx11_0#54]
Hi! someone can help me?

00000001	0.00000000	[2476] WinH264CreateDecoder	
00000002	0.00809860	[2476] [1210/182203.764:INFO:crash_reporting.cc(248)] Crash reporting enabled for process: renderer	
00000003	12.94585705	[6664] Proxy Dll process attach	
00000004	12.94688702	[6664] D3D12CreateDevice - Loaded d3d12_original.dll	
00000005	12.94702244	[6664] D3D12CreateDevice - LoadLibrary Result: 7ffa91430000	
00000006	12.94713783	[6664] D3D12CreateDevice - DEBUG: Checking Pix support	
00000007	12.94722366	[6664] D3D12CreateDevice - DEBUG: Attempt to load Pix support!	
00000008	12.95093155	[6664] D3D12CreateDevice - DEBUG: Pix WinPixGpuCapturer.dll not found, no extended Pix support available	
00000009	13.14042282	[6664] Proxy Dll process detach	
00000010	13.17901707	[9264] ExecCommandLine: "D:\Steam\steam.exe steam://run/1245620"	
00000011	13.17929745	[9264] ExecuteSteamURL: "steam://run/1245620"	
00000012	13.19441986	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp changed task to SynchronizingCloud with ""	
00000013	13.82475185	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp changed task to SiteLicenseSeatCheckout with ""	
00000014	13.82503033	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp changed task to CreatingProcess with ""	
00000015	13.82506275	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp waiting for user response to CreatingProcess ""	
00000016	13.82512760	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp continues with user response "CreatingProcess"	
00000017	13.84574223	[9264] Game process added : AppID 1245620 ""D:\Steam\steamapps\common\ELDEN RING\Game\start_protected_game.exe" -d3d11", ProcID 4408, IP 0.0.0.0:0	
00000018	13.85289955	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp changed task to WaitingGameWindow with ""	
00000019	13.89215660	[9264] GameAction [AppID 1245620, ActionID 21] : LaunchApp changed task to Completed with ""	
00000020	13.96581936	[4408] Proxy Dll process attach	
00000021	13.96712112	[4408] D3D12CreateDevice - Loaded d3d12_original.dll	
00000022	13.96720982	[4408] D3D12CreateDevice - LoadLibrary Result: 7ffa91430000	
00000023	13.96725273	[4408] D3D12CreateDevice - DEBUG: Checking Pix support	
00000024	13.96731663	[4408] D3D12CreateDevice - DEBUG: Attempt to load Pix support!	
00000025	13.97090435	[4408] D3D12CreateDevice - DEBUG: Pix WinPixGpuCapturer.dll not found, no extended Pix support available	
00000026	14.25822353	[4408] [S_API] SteamAPI_Init(): Loaded 'D:\Steam\steamclient64.dll' OK.	
00000027	14.32437801	[9264] Game process updated : AppID 1245620 ""D:\Steam\steamapps\common\ELDEN RING\Game\start_protected_game.exe" -d3d11", ProcID 4408, IP 0.0.0.0:0	
00000028	14.39016914	[4408] Shutdown handler: initalize.	
00000029	14.50969410	[4408] D3D12CreateDevice - Proxy function called!	
00000030	14.50972176	[4408] D3D12CreateDevice - Original function address available	
00000031	14.54892445	[4408] D3D12CreateDevice - CallResult: 0	
00000032	14.54895878	[4408] D3D12CreateDevice - CallResult indicates SUCCESS (S_OK)	
00000033	26.09385872	[9264] Game process removed: AppID 1245620 ""D:\Steam\steamapps\common\ELDEN RING\Game\start_protected_game.exe" -d3d11", ProcID 4408 	
00000034	26.09428787	[9264] Game 1245620 created interface STEAMAPPS_INTERFACE_VERSION008 / 	
00000035	26.09434891	[9264] Game 1245620 created interface SteamController008 / 	
00000036	26.09440231	[9264] Game 1245620 created interface SteamInput002 / 	
00000037	26.09445190	[9264] Game 1245620 created interface SteamUser021 / User	
00000038	26.09450722	[9264] Game 1245620 created interface SteamUtils010 / 	
00000039	26.09455490	[9264] Game 1245620 method call count for IClientUser::GetSteamID : 1	
00000040	26.09460640	[9264] Game 1245620 method call count for IClientUser::BIsSubscribedApp : 1	
00000041	26.09466171	[9264] Game 1245620 method call count for IClientUtils::GetAppID : 9	
00000042	26.09470940	[9264] Game 1245620 method call count for IClientUtils::RecordSteamInterfaceCreation : 7	
00000043	26.09476280	[9264] Game 1245620 method call count for IClientApps::GetAppData : 1	
00000044	26.09478951	[9264] Game 1245620 method call count for IClientAppManager::BIsDlcEnabled : 1	
00000045	26.09484291	[9264] Game 1245620 method call count for IClientAppManager::GetCurrentLanguage : 1	
00000046	26.09497833	[9264] Uploaded AppInterfaceStats to Steam	

[DebugView - Elden Ring log 2022-12-10.LOG](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/10200619/DebugView.-.Elden.Ring.log.2022-12-10.LOG)

**Comment by tuffee88** [comment]
As your card uses the Maxwell architecture (GM107 IC, according to Google) this should work as expected (we do have success reports from other Maxwell cards which have weaker specs if I remember correctly). 

 Can you try to start the game directly (not using the steam launcher) and comment on what is happening on screen (you do get past the white screen of death, right ? ) ?

**Comment by RenatoPassos** [comment]
@tuffee88 and @vernekotthonxdddd: my information might be outdated but, sometimes, laptop versions of GPUs gave weird/random problems to games and the OP is using a gtx 960_m_ . Could this be the case? This fix runs just fine in my 10 years old GTX 670...

**Comment by RenatoPassos** [comment]
Hmmm..... Just noticed a thing:

00000004 12.94688702 [6664] D3D12CreateDevice - Loaded d3d12_original.dll

@vernekotthonxdddd are you using the latest version of this fix? If that's the case, steps 2 and 3 of the instructions (https://github.com/tuffee88/d3d12ProxyEdrDx11_0#how-to-try-this-out-) have become optional. Maybe you could try the simplest setup possible and try @tuffee88's advice and see what happens?

**Comment by lowkeyrenato** [comment]
> As your card uses the Maxwell architecture (GM107 IC, according to Google) this should work as expected (we do have success reports from other Maxwell cards which have weaker specs if I remember correctly).
> 
> Can you try to start the game directly (not using the steam launcher) and comment on what is happening on screen (you do get past the white screen of death, right ? ) ?

I downloaded the cracked version too, (v1.02) and the game still not works:( the same problem happening: white screen and crash. But if i use your fix files this happens: white screen, after few seconds switchs to black screen and crash the same way (sorry for my english is so bad, im from hungary)

**Comment by lowkeyrenato** [comment]
> Hmmm..... Just noticed a thing:
> 
> 00000004 12.94688702 [6664] D3D12CreateDevice - Loaded d3d12_original.dll
> 
> @vernekotthonxdddd are you using the latest version of this fix? If that's the case, steps 2 and 3 of the instructions (https://github.com/tuffee88/d3d12ProxyEdrDx11_0#how-to-try-this-out-) have become optional. Maybe you could try the simplest setup possible and try @tuffee88's advice and see what happens?

Yes i use the release 3 (first i tried with the release 1 and not worked, then i reinstalled the game and tried with the release 3 too (still not worked))

Btw fun fact: my name is Renato too:) (in hungarian: Renátó)

**Comment by keerin** [comment]
> > Hmmm..... Just noticed a thing:
> > 00000004 12.94688702 [6664] D3D12CreateDevice - Loaded d3d12_original.dll
> > @vernekotthonxdddd are you using the latest version of this fix? If that's the case, steps 2 and 3 of the instructions (https://github.com/tuffee88/d3d12ProxyEdrDx11_0#how-to-try-this-out-) have become optional. Maybe you could try the simplest setup possible and try @tuffee88's advice and see what happens?
> 
> Yes i use the release 3 (first i tried with the release 1 and not worked, then i reinstalled the game and tried with the release 3 too (still not worked))
> 
> Btw fun fact: my name is Renato too:) (in hungarian: Renátó)

I'm sorry to see you are still having this issue Renato. I've also got a 960m and I've been using this fix since it appeared.

I updated the game recently and (as expected) it broke my game again, except this time I couldn't use any ashes of war effects and I had a floating green box that said Eliminations 99+ and every enemy had a green box beside their damage bar. Maybe more stuff was broken too but I noped out at that point.

Anyway, I just used this package and method again (skipped the optional steps). All I did was paste in the new .dll, rename the start_protected_game.exe to start_protected_game_original.exe and renamed the Elden Ring executable start_protected_game.exe. 

Since we're using the same card and the fix worked for me, let me know if I can provide any info that might help you.

**Comment by queenxlily** [comment]
i'm a bit late but don't matter, i tried keerin's fix and worked. Had to put the .dll and change the game's .exe to start_protected_game.exe, i used it on elden ring nightreign on a 940M. Didn't play the game yet but let's hope no more crashes xd, how unfortunate i can't play online with this fix
